php/ext/google/protobuf/json_options.h

```diff
@@ -0,0 +1,22 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#ifndef PHP_PROTOBUF_JSON_OPTIONS_H_
+#define PHP_PROTOBUF_JSON_OPTIONS_H_
+
+// Define constants
+#define JSON_SERIALIZE_OPTIONS_EMIT_DEFAULTS "EMIT_DEFAULTS"
```

Comment:
This flag was recently made consistently named in C++, Python and Java (also more verbose for clarity about what it affects), so it would be better to match here too:

JSON_SERIALIZE_OPTIONS_ALWAYS_PRINT_FIELDS_WITH_NO_PRESENCE